package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "twitter",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/arpit1107/twitter.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/arpit1107/twitter/issues"
+  },
+  "homepage": "https://github.com/arpit1107/twitter#readme",
+  "dependencies": {
+    "express": "^4.17.2"
```

Comment:
Please check the stable express version

Reply:
this is the latest stable version of express